keadm/app/cmd/util/common.go

```diff
@@ -40,6 +40,9 @@ const (
 
 	KubernetesDownloadURL = "https://apt.kubernetes.io/"
 	KubernetesGPGURL      = "https://packages.cloud.google.com/apt/doc/apt-key.gpg"
+	
+	KubernetesImageRepository = "k8s.gcr.io"
+	KubernetesPodNetworkCidr  = "10.244.0.0/16"
```

Comment:
I think we should use this block 100.64.0.0/10 as default.

Reply:
Yes, you are right, I will fix this, and recreate PR.